### ISSUE TITLE
Updates for references

### ISSUE THREAD

**Issue by yeeus** [YingZhou001/Immuannot#10]
Thanks the author for providing a nice tool! I have seen frequent updates for IPD-IMGT/HLA databases but not for Immuannot (until now it's release 3.58.0 while the author provided was 3.55.0). Can you give us the method to update the references on ourselves? I think that will be helpful for users who want to use the latest database version.
Thanks.

**Comment by YingZhou001** [comment]
Thanks for the comment. Please check the folder "prepare-reference" for how I prepare reference based on the latest IMGT database. Please let me know if anything wrong.

**Comment by yeeus** [comment]
Thank you for quickly update!

**Comment by yeeus** [comment]
Because of five HLA-DRB genes (DRB2,6,7,8,9) were not in IPD-IMGT/HLA database release 3.55.0 but are in release 3.58.0, I have tried with the latest version to capture these genes. However, in some reasons (it seems due to incomplete `UTR`s), these five genes were tagged as `goodTemplate=False` and partial. Therefore in the last, these genes were not annotated by Immuannot. And I used the old version which I had applied to all samples. Still thanks!
